### PR TITLE
Fix replication list images - With fixed test

### PR DESCRIPTION
**Pull request by MrMEEE**

### CODE REVIEW

**Comment by MrMEEE** [comment]
@bitsf did all the work :)

File: src/common/utils/registry/repository.go [hunk 1]
```diff
@@ -69,49 +69,61 @@ func parseError(err error) error {
...
+			tagsResp := struct {
+				Tags []string `json:"tags"`
+			}{}
+
+			if err := json.Unmarshal(b, &tagsResp); err != nil {
+				return tags, err
+			}
+
+			tags = append(tags, tagsResp.Tags...)
+			// Link: </v2/_catalog?last=library%2Fhello-world-25&n=100>; rel="next"
```
Comment (ywk253100) [inline src/common/utils/registry/repository.go]: You mean `/v2/<name>/tags/list`?

Comment (ywk253100) [reply src/common/utils/registry/repository.go]: Please update the comment here

Comment (MrMEEE) [reply src/common/utils/registry/repository.go]: @bitsf Could you comment on this??

Comment (bitsf) [reply src/common/utils/registry/repository.go]: yes, my typo, this comment should be
// Link: </v2/library/hello-world/tags/list?n=100>; rel="next"
// Link: <http://domain.com/v2/library/hello-world/tags/list?n=100>; rel="next"

Comment (bitsf) [reply src/common/utils/registry/repository.go]: actually the url is as below
'<https://444571669415.dkr.ecr.ap-northeast-1.amazonaws.com/v2/test/nginx-photon/tags/list?last=ukD72mdD%2FmC8b5xV3susmJzzaTgp3hKwR9nRUW1yZZ4R3Jc13zaALoZQP9ut3IYattOaKlXMRmYYgRHibI9VTKT2qZ%2BXD7PV4LjLE6OjHmQkRwXypd7HXj%2FSnA9iMm3YvNsd4LmPQrO4fpYZgnZZ8rzIIYqex6%2B3A3%2FmKcTsNKkkKwRror1sjGrDqkw6MRYRWLUC5ZAd%2B7VjpfVsS5HLLEZltW%2Fl3aYbulKYdpdl0Z46tIb6CcBgfvi1ECZhEkEGPnTgU%2B5oeI8mpMftGK6i3j6U6Vn0rYXucTE2coh%2BlSCEpI6Nkzh%2F1RptJZioNqEf24qDS8t0eYzqJ%2B1%2BtWGyBvnC1qYWEFgixqfhcN8hb1g%3D>; rel="next"'

Comment (bitsf) [reply src/common/utils/registry/repository.go]: the two comment lines should both be changed. use the simple example is enough.

// Link: </v2/library/hello-world/tags/list?last=......>; rel="next"
// Link: \<http://domain.com/v2/library/hello-world/tags/list?last=......>; rel="next"

Comment (MrMEEE) [reply src/common/utils/registry/repository.go]: ok, done :)

**Comment by ywk253100** [comment]
@MrMEEE Thanks. Please squash all your commits into single one

**Comment by MrMEEE** [comment]
@ywk253100 I'm not very good at git.. how do I do that??

**Comment by ywk253100** [comment]
@MrMEEE Please refer to https://github.com/wprig/wprig/wiki/How-to-squash-commits

**Comment by MrMEEE** [comment]
I tried, but failed?

**Comment by bitsf** [comment]
what's the error do you meet.

actually squash also can be done after review, when click "Merge pull request".

**Comment by MrMEEE** [comment]
I just tried to follow the guide.. and just ended up with 10 commits instead of 5?

**Comment by bitsf** [comment]
You should give more details about your problem.
The target is you should just have one commit here.

**Comment by MrMEEE** [comment]
#10083 Should be good...

**Pull request closed without merging**